src/react-styleguide/src/components/Icons/Icons.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+
+const Icon = ({color, icon}) => {
+  const commonClasses = 'absolute br-pill flex justify-center items-center eh-input-status-icon'
+    
+  return (
+    <div className={`${commonClasses} ${color}`}>
+      <i className={`fa ${icon}`} />
+    </div>
+  )
+}
+
+export const SuccessIcon = () => <Icon color='green' icon='fa-check-circle' />
+export const PlusIcon = () => <Icon color='blue' icon='fa-plus-circle' />
+export const MinusIcon = () => <Icon color='light-gray' icon='fa-minus-circle' />
+export const XIcon = () => <Icon color='red' icon='fa-times-circle' />
+export const BangIcon = () => <Icon color='red' icon='fa-exclamation-circle' />
+export const QuestionIcon = () => <Icon color='yellow' icon='fa-question-circle' />
```

Comment:
Could this be `purpose` instead of color? Does each color serve a purpose? Also, would be nice if we used the same color names everywhere and declared these in their own module to consume across components.